appIos/post/post/PostScreen.swift

```diff
@@ -69,7 +69,9 @@ private struct PostState: View {
                             alignment: .leading,
                             spacing: 8
                         ) {
-                            PostHeader(header: post.raw.headline ?? "--")
+                            PostHeader(
+                                header: post.raw.headline ?? "--",
```

Comment:
Is this cast necessary here?

Reply:
This is Xcode complaining that `KotlinInt` cannot be passed as `Int`
<img width="527" alt="Screenshot 2022-04-11 at 07 49 28" src="https://user-images.githubusercontent.com/5462605/162680742-f410d40d-cee7-40b4-8d19-4be995ce9b23.png">

Reply:
Another posh Apple issue? :)

Reply:
I see, these 2 guys copying each other with the nullability. I'll apply this one